.github/workflows/release.yaml

```
@@ -14,39 +14,93 @@
 
 name: Release
 on:
-  push:
-    tags:
-      - '**'
+  - push
+  - pull_request
 jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        id:
+          - "rolling"
+          - "kilted"
+          - "jazzy"
+          - "humble"
+    steps:
+      - uses: actions/checkout@v5
+      - name: Build URDF files
+        run: |
+          docker container run \
+            --rm \
+            --volume ${PWD}:/build \
```

It's a source directory:

```suggestion
            --volume ${PWD}:/source \
```